### PR TITLE
Features/fixes

### PR DESCRIPTION
**Pull request by zakrad**

### CODE REVIEW

File: test/7-FixLender.test.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        .withArgs(addresses[1], 2 * amount);
+      const bonusAfterWith = await bonusToken.balanceOf(addresses[1]);
+      const stableAfterWith = await stableToken.balanceOf(addresses[1]);
+      const bonusBalance = bonusAfterWith.sub(bonusBeforeWith);
+      const stableBalance = stableAfterWith.sub(stableBeforeWith);
+      const actualBonus = parseFloat(await fromBonus(bonusBalance));
+      const actualStable = parseFloat(await fromStable(stableBalance));
+      expect(actualBonus).to.be.equal(expectedBonus);
+      expect(actualStable).to.be.equal(expectedStable);
+      console.log(`withdrawn ${actualStable} stable for account 1`);
```
Comment (Aboudjem) [inline test/7-FixLender.test.js]: remove the console.log

File: test/7-FixLender.test.js [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+        PoolMaxLimit2,
+        false
+      );
+      await lenderContract.deployed();
+      await lenderContract.switchStrategy(strategy.address);
+      await strategy.grantRole(LenderPoolAccess, lenderContract.address);
+      const initialDeposit = await toStable("10000");
+      await stableToken.approve(lenderContract.address, initialDeposit);
+      await lenderContract.deposit(initialDeposit);
+      console.log("account 0 deposited 10k");
```
Comment (Aboudjem) [inline test/7-FixLender.test.js]: remove the console.log

File: test/7-FixLender.test.js [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+    it("Should withdraw all rewards + principal after pool end date", async function () {
+      const amount = await toStable("100");
+      const bonusAmount = await toBonus("100");
+      await stableToken.transfer(addresses[1], amount);
+      await bonusToken.transfer(lenderContract.address, bonusAmount);
+      await stableToken
+        .connect(accounts[1])
+        .approve(lenderContract.address, amount);
+      await lenderContract.connect(accounts[1]).deposit(amount);
+      console.log("account 1 deposited 100");
```
Comment (Aboudjem) [inline test/7-FixLender.test.js]: remove the console.log

File: test/7-FixLender.test.js
Comment (Aboudjem) [inline test/7-FixLender.test.js:1]: remove the console.log s

**Review by Aboudjem — APPROVED** [review]

**Review by Aboudjem — APPROVED** [review]

**Review by khawlahssn — APPROVED** [review]

**Pull request merged**